src/Bicep.Core/Emit/ScopeHelper.cs

```diff
@@ -418,7 +418,7 @@ void logInvalidScopeDiagnostic(IPositionable positionable, ResourceScope supplie
                     if (resource.TryGetScopeSyntax() is { } scopeSyntax)
                     {
                         // it doesn't make sense to have scope on a descendent resource; it should be inherited from the oldest ancestor.
-                        diagnosticWriter.Write(scopeSyntax, x => x.ScopeUnsupportedOnChildResource(ancestors.Last().Resource.Symbol.Name));
```

Comment:
>ancestors.Last().Resource.Symbol.Name [](http://example.com/codeflow?start=99&length=37)

This was unused, but I am wondering if someone forgot to update the error message to actually consume the value.

Reply:
Would you want to create an issue for this?

Reply:
Hmm. Maybe it's not worth it since we never included the info in the error and didn't get any complaints about it. I'm ok either way. Up to you.